### ISSUE TITLE
[Bug]: egress routing problem on win10/11 clients in same local range

### ISSUE THREAD

**Issue by bpmarine** [gravitl/netmaker#1867]
### Contact Details

bpmarine@gmail.com

### What happened?

I have a persistent Windows 10/11 netclient with local egress gateway routing issue.  I have tested in different locations with completely different netmaker servers.

Since version 0.16.0 Netclients on Windows Desktops work fine until an egress gateway is added on the local network giving access to their network range. They continue to function over wireguard, but they can no longer make HTTPS or SHH connections to the Netmaker server, or to other nodes running netclient such as a VPS webserver. Disabling the egress gateway or disconnecting the node allows normal comms to continue, e.g. opening the dashboard, connecting to websites on a full client connected webserver, making SSH connections to other nodes using their public IP. 

None of the Windows Server 2016 nodes are affected by this, or any of the linux nodes in the same network range. All continue to route traffic as expected. I have tried adding win 10/11 clients via the CLI as per the WinServer nodes, but it makes no difference.

At v 0.17.0 I scrubbed my server and setup a fresh VM just in case, but same issues. v0.15 was the last version that worked as expected for me.

The affected nodes cannot make pull requests either, with the timeout error below.

### Version

v0.17.1

### What OS are you using?

Windows

### Relevant log output

```shell
Post "https://api.netmaker.my-domain.org:443/api/nodes/adm/my-net/authenticate": dial tcp xx.xx.x.xx:443: connectex: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.
```

### Contributing guidelines

- [X] Yes, I did.

**Comment by taladar** [comment]
This sounds similar to the problem that made me request #1836 as a feature.

Does your egress route include any IPs that would be involved in those connections that stop working?

If that is the case you probably have some unintended asymmetric routing (request without VPN, response with the VPN).

**Comment by bpmarine** [comment]
No, most of the nodes in these network ranges continue to route correctly, including all of the Linux based ones and also windows sever nodes. Only windows desktop 10/11 are not behaving. They used to function as expected version 0.8 to 0.15.
